backend/backend/Controllers/ReactionTimeController.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Mvc;
+
+namespace backend.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class ReactionTimeController : ControllerBase
+{
+
+    private readonly ILogger<ReactionTimeController> _logger;
+
+    public ReactionTimeController(ILogger<ReactionTimeController> logger)
+    {
+        _logger = logger;
+    }
+
+    [HttpGet(Name = "GetNumber")]
+    public ReactionTime Get()
+    {
+        Response.Headers.Append("Access-Control-Allow-Origin", "http://localhost:3000");
+
+        const int MILLISECONDS_IN_SECOND = 1000;
```

Comment:
Man atrodo metoduose cont kintamuosius reiktu ne vien didziosiomis rasyti.
Rider rekomendacija: Name 'MILLISECONDS_IN_SECOND' does not match rule 'Local constants'. Suggested name is 'millisecondsInSecond'.